src/main/java/org/jabref/gui/BasePanel.java

```diff
@@ -56,11 +56,7 @@
 import org.jabref.gui.specialfields.SpecialFieldDatabaseChangeListener;
 import org.jabref.gui.specialfields.SpecialFieldValueViewModel;
 import org.jabref.gui.specialfields.SpecialFieldViewModel;
-import org.jabref.gui.undo.CountingUndoManager;
-import org.jabref.gui.undo.NamedCompound;
-import org.jabref.gui.undo.UndoableFieldChange;
-import org.jabref.gui.undo.UndoableInsertEntry;
-import org.jabref.gui.undo.UndoableRemoveEntry;
+import org.jabref.gui.undo.*;
```

Comment:
Just a hint: This will create a check style error. Star imports are not allowed.
For intellij and eclipse we provide a code style configuration.
https://github.com/JabRef/jabref/tree/master/config